networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc

```diff
@@ -68,6 +68,12 @@ The following advisory is available for the AWS Load Balancer Operator version 1
 
 * link:https://access.redhat.com/errata/RHEA-2023:1954[RHEA-2023:1954 Release of AWS Load Balancer Operator on OperatorHub Enhancement Advisory Update]
 
+[IMPORTANT]
+====
+The AWS Load Balancer Operator version 1.x.x cannot upgrade automatically from version 0.x.x, the Technology Preview version.
```

Comment:
This is really just a style choice because I don't think you really need to say version twice, but you could say something like:
```suggestion
The AWS Load Balancer Operator version 1.x.x cannot upgrade automatically from the Technology Preview version 0.x.x.
```

Reply:
Of course!